.tekton/tasks/sync-argocd-app.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: app-sync
+spec:
+  description: >-
+    This task is for syncing the staging cluster argocd application
+  workspaces:
+   - name: argocd-config
+   - name: infra-deployment-git
+  steps:
+    - name: argocd-app-sync
+      image: quay.io/devtools_gitops/test_image:4.0.5
```

Comment:
The name of this image tells me we should probably not rely on it. Consider using a more "mainstream" image.

Reply:
What do you suggest?